File: public/examples/integrations/toolkits/search/youtube/get_youtube_video_details_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "Search.GetYoutubeVideoDetails"

tool_input = {
    "video_id": "dQw4w9WgXcQ",
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
)
print(response)

File: public/examples/integrations/toolkits/search/youtube/get_youtube_video_details_example_call_tool.js
import { Arcade } from '@arcadeai/arcadejs';

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = 'Youtube.GetYoutubeVideoDetails';

const toolInput = {
  video_id: 'dQw4w9WgXcQ',
};

const response = await client.tools.execute({
  tool_name: TOOL_NAME,
  input: toolInput,
});

console.log(response);
